types/parse-srcset/index.d.ts

```diff
@@ -0,0 +1,10 @@
+interface ParsedSrcset {
+    url: string;
+    d?: number;
+    h?: number;
+    w?: number;
+}
+
+declare function parseSrcset(value: string): ParsedSrcset[];
+
+export = parseSrcset;
```

Comment:
This is a UMD module, you probably also want `export as namespace parseSrcset`

Reply:
Thanks, good catch, I've updated.